web/js/codeworld.js

```diff
@@ -585,19 +642,27 @@ function run(hash, dhash, msg, error) {
     if (hash) {
```

Comment:
I believe that somewhere in `run`, you want to stop the current recording, if there is one?